### PR TITLE
Upstream Merge

### PR DESCRIPTION
**Pull request by ghost**
Closes #3567
Closes #3566
Closes #3566
Closes #3565
Closes #3564
Closes #3563
Closes #3542
Closes #3537
Closes #3525 
Closes #3569

### CODE REVIEW

**Review by j-mie — DISMISSED** [review]

File: hippiestation/code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm [hunk 1]
```diff
@@ -1,28 +0,0 @@
-/datum/clockwork_scripture/create_object/summon_arbiter
```
Comment (GuyonBroadway) [inline hippiestation/code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm:1]: **MIEN ROBOTÉR!!!**

Comment (KayeArray) [reply hippiestation/code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm:1]: Not as you are. But become the Arbiter, and you shall be set loose against this heresy with our blessing.

**Comment by GuyonBroadway** [comment]
I still lament the loss of my beloved neovgre.

**Comment by ghost** [comment]
Feel free to add it back in

**Review by GuyonBroadway — APPROVED** [review]

**Comment by ghost** [comment]
build this

**Pull request merged**